taskcluster_util/taskcluster_download.py

```diff
@@ -150,12 +151,15 @@ def run(self):
             # no artifact_name, then get the latest artifacts list
             self.show_latest_artifacts(task_id)
         elif self.should_display_signed_url_only is True:
-            print self.artifact_downloader.get_signed_url(task_id, self.artifact_name)
+            signed_url = self.artifact_downloader.get_signed_url(task_id, self.artifact_name)
+            print(signed_url)
+            return signed_url
```

Comment:
It's just the refactoring.
If other python code reuse this tc_download class, they can get the result.